src/App.tsx

```diff
@@ -15,47 +15,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [isReverse, setIsReverseAction] = useState(false);
+  const [sortType, setSortTypeAction] = useState(SortType.NONE);
+
+  const alphabet = () => {
```

Comment:
`handleSortByType`
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364